src/core/mavsdk_impl.cpp

```diff
@@ -282,8 +312,7 @@ uint8_t MavsdkImpl::get_own_system_id() const
 {
     switch (_configuration.load()) {
         case Mavsdk::Configuration::GroundStation:
-            // FIXME: This doesn't make much sense actually but it seems to work.
-            return 0;
+            return 254;
```

Comment:
I guess this should be: 

MAV_COMP_ID_MISSIONPLANNER

Reply:
Yes.